android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -1095,6 +1095,57 @@ public static BlockchainToken createEthereumBlockchainToken(String chainId) {
         return eth;
     }
 
+    public static BlockchainToken makeNetworkAsset(NetworkInfo network) {
+        String logo;
+
+// TODO: Add missing logos
```

Comment:
Is any issue for this?

Reply:
I just created one https://github.com/brave/brave-browser/issues/23934.

Reply:
thanks